lib/providers/services/esr_service.dart

```diff
@@ -0,0 +1,76 @@
+import 'dart:typed_data';
+
+import 'package:eosdart_ecc/eosdart_ecc.dart';
+import 'package:seeds/features/scanner/telos_signing_manager.dart';
+import 'package:seeds/providers/services/eos_service.dart';
+import 'package:seeds/screens/app/wallet/custom_transaction.dart';
+import 'package:eosdart/src/serialize.dart' as ser;
+import 'package:dart_esr/dart_esr.dart';
+
+class IdentityResponse {
+  final String callback;
```

Comment:
callback? 
a string? 

When i think of a callback i think of a `callback` i think of a function. 

Can we add comments to these params, explain what they represent.

Reply:
Callback is a callback URL provided by the identity request

Reply:
ah i see, perhaps name it 
callbackUrl